src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
+      return preg_replace('@((?:width|height)=.+?\s+)@', '', $matches[0]);
```

Comment:
3. Yes we should change this! Will need an extensive regex which will take long to finish when called via search-replace but this would be the best solution.